File: gaia-sdk-python/gaia_sdk/graphql/response/type/SkillPracticedImpulse.py



class SkillPracticedImpulse:
    """
    This impulse returns the result of a skill practice query request
    """
    def id(self) -> Uuid:
        return self.id
    """
    The result of the skill practice
    """
    def data(self) -> Struct:
        return self.data

File: gaia-sdk-javascript/src/graphql/response/type/SkillPracticedImpulse.ts


import {Uuid, Timestamp, Struct, Long} from "../../GaiaClient";

/**
* This impulse returns the result of a skill practice query request
*/
export interface SkillPracticedImpulse {
    id?:Uuid,
    /**
    * The result of the skill practice
    */
    data?:Struct
}
